pkg/model/ordered_dict.go

```diff
@@ -163,6 +163,25 @@ func UnorderedTypes(inter interface{}) interface{} {
 		}
 
 		return tab
+
```

Comment:
Can we factorize code for all slice branches ?

Reply:
Without reflect i don't think we can...